docs/source/technical_tutorials/access/accessing_LVIS_data.ipynb

```diff
@@ -0,0 +1,1765 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    filename = os.path.join(dataDir, "LVISF1B_US2018_1107_R2011_067463.h5")
</pre><p>Same comment, for each dataset in this notebook, I think that we could remove this hard coded value and instead use the output of the cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Sure. </p>